src/Components/WebView/WebView/src/Services/WebViewRenderer.cs

```diff
@@ -0,0 +1,95 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Components.RenderTree;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNetCore.Components.WebView.Services
+{
+    internal class WebViewRenderer : Renderer
+    {
+        private readonly Queue<UnacknowledgedRenderBatch> _unacknowledgedRenderBatches = new();
+        private readonly Dictionary<string, int> _componentIdBySelector = new();
+        private readonly Dispatcher _dispatcher;
+        private readonly IpcSender _ipcSender;
+        private long nextRenderBatchId = 1;
```

Comment:
Do we need this here?

For Blazor Server I can understand, but for Blazor WebView I think we can rely on a simple ACK without having to keep track of the id

Reply:
It just helps us discover more quickly if we have any bugs related to out-of-order delivery.

Reply:
How can this situation arise?

Reply:
We can have bugs :)

Reply:
BTW in my experience I've never seen such a bug while working on MBB and haven't seen it from customers, but perhaps it can more easily be an issue _while we develop_ the code, so it seems like a good precaution. I cannot imagine such a bug would be easy to debug when things simply explode and behave oddly.